.github/workflows/nightly-wheel-build.yaml

```diff
@@ -18,13 +22,6 @@ permissions:
 jobs:
   call-workflow-build-wheels:
     uses: ./.github/workflows/wheels-recipe.yaml
-    with:
-      CIBW_ENVIRONMENT: >
-        PIP_PRE=1
-        PIP_EXTRA_INDEX_URL="https://pypi.anaconda.org/scientific-python-nightly-wheels/simple"
```

Comment:
We used to build our nightlies based on upstream nightlies. Do we want to continue doing so?

Reply:
We think this is a relic from the NumPy 1 to 2 transition.